Added test in commands_test.go

sula7 opened the pull request:
Added test to method `TestChangeGroupTimeZone` in **commands_test.go**
#54

Commit:
#54 Added test for func TestChangeGroupTimeZone in commands_test.go

anatoliyfedorenko commented on bot/commands.go:
```diff
@@ -789,10 +789,10 @@ func (b *Bot) ChangeGroupTimeZone(event tgbotapi.Update) (string, error) {
-	tz := event.Message.CommandArguments()
+	tz := strings.TrimSpace(event.Message.CommandArguments())
 
 	if strings.TrimSpace(tz) == "" {
```
убери вот здесь TrimSpace раз уж вверху сделал уже

sula7 replied:
@anatoliyfedorenko resolve

Commit:
Corrected mistakes in func ChangeGroupTimeZone

1. Mistake was in Length of MessageEntity, I used 8 instad of 9, missing whitespace in file commands_test.go 2. According to PR, removed TrimSpace() in commands.go Now, tests pass
